scripts/cc-cli/src/utils/tx.ts

```diff
@@ -28,21 +28,28 @@ export async function signSendAndWatch(
         unsubAndResolve(result);
       }
       if (dispatchError) {
+        let blockHash: string | null = null;
+        if (status.isInBlock) blockHash = status.asInBlock.toHex();
```

Comment:
Q1) How can something be in a block and failed at the same time ?

Reply:
when a transaction is accepted in the queue it can fail its called an extrinsic failure, and it will be in the chain state, and you would have lost the CTC fee, same on Ethereum.

Reply:
Hmm, I thought only successful transactions make it into a block.